generators/github/USAGE

```diff
@@ -0,0 +1,3 @@
+Description:
+	Creates a bunch of `.github/` files:
+	Issue & PR templates, CODE_OF_CONDUCT, CONTRIBUTING and SECURITY.md.
```

Comment:
Newline missed. 

I wonder always about why was this newline thing... seems related to the way POSIX considers a line.

https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

![Screenshot 2022-01-12 at 18 13 03](https://user-images.githubusercontent.com/5908855/149188690-906b14c4-25e9-4811-813a-99db8f492e8e.png)